handbook/ce/project-work.md

```diff
@@ -0,0 +1,50 @@
+ - **Project Owner**: An individual who is responsible for the successful outcome of the project. They may delegate work to Project Contributors, but are responsible for Stakeholder alignment, timely progression of the project, keeping the Project Brief up-to-date, and updating status of the project in Asana
+ - **Project Contributor**: An individual who works on the project.
+ - **Project Stakeholder**: An individual who either provides input into the project and/or is notified of progress of the project.
+ - **Project Impact**: Defines the impact the project has on either Sourcegraph, Sourcegraph's customers, or both. The possible values are as follows:
+   1. The project will affect *most customers* and *multiple Sourcegraph departments*
+   2. The project will either affect *most customers* or *multiple Sourcegraph departments*
+   3. The project will affect a *small number of customers*
+   4. The project will affect a *single Sourcegraph department*
+
+## Process
+
+ - Create a copy of the [project brief](https://docs.google.com/document/d/1xImkyvsZkwnZ2OtqTl6ocsaw8YBPSQwfGWHraIZLIyM/edit?usp=sharing)
+   - fill in at least the mandatory fields (Title, Purpose, Objective)
+   - save the document under [Sourcegraph -> Customer Engineering -> Project Briefs](https://drive.google.com/drive/folders/19VGpxb2wZEswMOvwJSoOK8MEV492TU0M) folder with the name that follows the pattern: ```[[Title]] - CE Project Brief```
+ - Create a new [Asana](https://app.asana.com/0/1200768248900861/list) task (under the ```CE Projects``` project) tracking the project proposal
+   - Ensure the task title matches the brief title
+   - If you intend on being the Project Owner, assign yourself as the "Assignee"
+   - Select the appropriate "Project Type"
+     - If your project does not adequately fit into an existing "Project Type", leave it blank, and raise this at the next "CE Project Planning & Review meeting"
+   - Optionally provide a "Project Impact"
+   - Set the status of the task to "Proposed"
+   - Save the URL of the Project Brief
+ - Add a new entry in the CE Project Planning & Review [agenda document](https://docs.google.com/document/d/1xTq5Tj_y9cV9aDIhQXWj2QN9Z-gbWhvtkCVAk4b0LD0/edit?usp=sharing) as follows:
+   - ```Review proposed project [[link to brief]]```
+   - If you need to suggest a new Project Type, append: ```and recommend project type```
+ - At the next CE Project Planning & Review meeting, the proposal will be discussed, and the following needs to be agreed:
+   - Relative priority of the project and approximate timelines
+   - Project owner, if not already assigned
+   - Relevant stakeholders and their involvement
+   - Project contributors, and their roles
+   - Optionally (and time depending) other fields may be discussed with the wider team
+ - The Project Owner updates the status of the project in Asana to "Agreed" and then, once work starts, to "In Progress"
```

Comment:
Do we want to change the process so that this is just changed in the meeting? One less step, seems like.